src/rust/ide/view/src/documentation.rs

```diff
@@ -154,6 +152,22 @@ impl Model {
         }
     }
 
+    /// Fixes indentation in the first line of documentation by looking at next line, as documentation
+    /// doesn't know what was its indentation in the Enso source file.
```

Comment:
no, putting fixed amount of spaces wont fix the issue.
Take a look at some enso code:
```py
type foo
    type bar
        ## docs here
           docs there
        type baz
```
We need to add the indentation of whole block plus the indent of 2 hashes - that is in this case 10 additional spaces, otherwise doc parser will get code like this:
```
 docs here
          docs there
``
and will create a synopsis with raw text "docs here" and code block of  "docs there".

As we talked on dm, if there is only one line, there is no problem, it just wont do anything, and will work. If the next line is empty, there is still no problem, as what is in the next box will just become new paragraph, just as intended, in "body" section. 

The only corner case is when the user specifically intends to make second line a code block, but that must be a really rare case, as i havent seen such in any doc in stdlib.

To sum up, this thing is really not a trivial fix as the doc code we get doesnt know anything about its origin, what was its indentation etc, hence the need to write such a hack. To be further discussed with @wdanilo but I dont see any other reasonable solution currently.

Reply:
After talk I've removed hack method, this PR wont fix indent in first line.